kserve-rest-proxy.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: kserve-rest-proxy
+  version: 0.12.0
+  epoch: 0
+  description: "REST API proxy for kserve, the standardized serverless ML inference platform on Kubernetes"
+  copyright:
+    - license: Apache-2.0
+
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kserve/rest-proxy
+      tag: v${{package.version}}
+      expected-commit: c5a4f74b894506b058469eadcbf17ea456145ff9
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./proxy
+      output: rest-proxy
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
```

Comment:
you only need curl if you're going to check it. which is good to do , but you're not doing that right now.

Reply:
TY, missed removing that once I realized health checking wouldn't be as trivial as a `curl` request.